### ISSUE TITLE
lint: remove trailing period from flagDeref checker summary

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#678]
Summary field should not contain trailing dot.
```go
d.Summary = "Detects immediate dereferencing of `flag` package pointers."
```
